lib/phobos_db_checkpoint/tasks.rb

```diff
@@ -11,7 +11,7 @@ module Tasks
 
       task :load_config do
         PhobosDBCheckpoint.load_db_config
-        task_db_config = Hash[PhobosDBCheckpoint.env, PhobosDBCheckpoint.db_config.merge('pool' => 1)]
+        task_db_config = Hash[PhobosDBCheckpoint.env, PhobosDBCheckpoint.db_config]
```

Comment:
The logic for removing this is that we should just use whatever we have configured, no need to set something explicitly which may override `pool` for no particular reason.